deploy/operator_ironic.yaml

```diff
@@ -117,13 +168,13 @@ spec:
                 configMapKeyRef:
                   name: ironic-bmo-configmap
                   key: http_port
-            - name: IP
+            - name: PROVISIONING_INTERFACE
               valueFrom:
                 configMapKeyRef:
                   name: ironic-bmo-configmap
-                  key: http_ip
+                  key: provisioning_interface
         - name: ironic
```

Comment:
Side note: we're splitting this container, see https://github.com/metal3-io/ironic-inspector-image/pull/26 and https://github.com/openshift-metal3/dev-scripts/pull/646

Reply:
The change has merged, could you actually do the split please?